Documentation/ApiOverview/ContentElements/CreatePlugins.rst

```diff
@@ -10,14 +10,14 @@ How to create plugins with the Extbase framework and Fluid templating engine
 is handled in depth in the chapter
 :ref:`extbase_registration_of_frontend_plugins`.
 
-There are basically three ways to create plugins in TYPO3:
+There are basically two ways to create frontend plugins in TYPO3:
 
 #.  With the Extbase framework using :php:`configurePlugin()` in the file
     :file:`ext_localconf.php` and :php:`registerPlugin()` in the file
     :file:`Configuration/TCA/Overrides/tt_content.php`
-#.  Create a frontend plugin using Core functionality and a custom controller
+#.  Create a "General Plugin" using Core functionality and a custom controller
```

Comment:
```suggestion
#.  Create a plugin using Core functionality and a custom controller
```
I would stay with just "plugin" here, it doesnt have to be a general plugin, it can also be a content elemern

Reply:
The topic is "Create plugins". Content element is the category above plugins. Here it should be more specific. 
Garvin Hicking has proposed to use the term "General plugin" as it is used in the TYPO3 backend.

Reply:
you are misunderstanding something. We are talking about apple and pears. In this context it is the implementation of the controller Extbase vs Core functionality. In https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/pull/4458 we are talking about how to save it to the database and what cType to use

Reply:
No, we are talking about "Extbase plugin" versus "General plugin".  Just look at the chapter of the page.
On the other page we are talking about the same 2 possible variants.